### ISSUE TITLE
Implement `.update(dict_values)` on `Skeleton`/`SkeletonInstance`

### ISSUE THREAD

**Issue by sveneberth** [viur-framework/viur-core#1098]
provide a `dict` or kwargs to merge multiple values into skel (like [`dict.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update))

**Comment by phorward** [comment]
> provide a `dict` or kwargs to merge multiple values into skel (like `dict.update()`)

More context, please!

**Comment by sveneberth** [comment]
Basically it's this method which should be added into `SkeletonInstance`:
```py
    def update(self, *args, **kwargs) -> None:
        for key, value in dict(*args, *kwargs).items():
            self[key] = value    
```

So you can write:
```py
skel.update(name="foo", sortindex=2)
# or
skel.update({"name": "foo", "sortindex": 2})
```

But this needs to be tested and we should think about if this should use `setBoneValue` and then we have to ensure this works for every bone, ...

**Comment by phorward** [comment]
I think this should make use of the `__or__`-operator as well, so one can write
```py
skel |= {"name": "foo", "sortindex": 2}
```

**Comment by ArneGudermann** [comment]
Closed by #1103
